### PR TITLE
EV-82 Test Footer component and emailService

### PR DESCRIPTION
**Pull request by Danyil-man**
### Jira

* [EV-52](https://chita.atlassian.net/browse/EV-52)

### Description
Created test for Footer component and send button.
Also, created test for emailService which in the Footer component for subscribe with email.

### Summary of change
Fix errors and warnings.

### Checklist

- [x] Includes a link to JIRA ticket and all related PR (API dependencies etc.)
- [x] Includes a brief description of what has been changed
- [x] Test coverage for **new** code doesn't decrease the existing test coverage
- [x] Lint errors/warnings fixed
- [x] New code is documented if necessary
- [x] Dev Testing has been successfully finished
- [x] Reviewed by 2 developer on your team
- [x] Reviewed by 1 expert

### CODE REVIEW

File: src/services/emailService.test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    expect(correctEmail).toStrictEqual({});
+  });
+  it("Email without at(@)", () => {
+    const inCorrectEmail = isValidEmail("examplegmail.com");
+    expect(inCorrectEmail).toStrictEqual({ email: error });
+  });
+  it("Email without address", () => {
+    const inCorrecAddress = isValidEmail("examplegmailcom");
+    expect(inCorrecAddress).toStrictEqual({ email: error });
+  });
```
Comment (MelenkoOleksandr) [inline src/services/emailService.test.js:21]: Why you test isValidEmail func in this file? You need to create tests only for emailService. Remove them or create new file for this tests.

**Review by MelenkoOleksandr — COMMENTED** [review]
Fix this

**Review by LisovyiNazar — DISMISSED** [review]

File: src/components/Footer/footer.test.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import ReactDOM from "react-dom";
+import { create } from "react-test-renderer";
+import Footer from "./Footer";
+
+describe("test Footer", () => {
+  it("check Footer component", () => {
+    const div = document.createElement("div");
+    ReactDOM.render(<Footer />, div);
+    ReactDOM.unmountComponentAtNode(div);
+  });
```
Comment (andrii-kuzmichov) [inline src/components/Footer/footer.test.js:11]: What do you check here?

File: src/components/Footer/footer.test.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  it("check Footer submit button that its not empty", () => {
+    const component = create(<Footer />);
+    const instance = component.root;
+    const button = instance.findByType("button");
+    expect(button.children.length).toBe(1);
+  });
```
Comment (andrii-kuzmichov) [inline src/components/Footer/footer.test.js:17]: Do you check for all Footer component just count of button?

File: src/services/EmailService.js [hunk 4]
```diff
@@ -1,12 +1,18 @@
...
+      "service_hxod138",
+      "template_1k257d8",
+      values,
+      "hIa6EbQItOhL_Sxmg",
+    );
+    return true;
+    // eslint-disable-next-line no-else-return
+  } else {
+    return false;
+  }
```
Comment (andrii-kuzmichov) [inline src/services/EmailService.js]: it looks weird

File: src/services/emailService.test.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import emailService from "./EmailService";
+import isValidEmail from "../components/helpers/validators/email-address-validator";
```
Comment (andrii-kuzmichov) [inline src/services/emailService.test.js]: why you check here method from diff file?

**Review by andrii-kuzmichov — DISMISSED** [review]

File: src/components/Footer/footer.test.js (hunk 2 above)
Comment (Danyil-man) [reply src/components/Footer/footer.test.js:11]: Test for component

File: src/components/Footer/footer.test.js (hunk 3 above)
Comment (Danyil-man) [reply src/components/Footer/footer.test.js:17]: In this case, yes

File: src/services/EmailService.js (hunk 4 above)
Comment (Danyil-man) [reply src/services/EmailService.js]: it checks for email.test

File: src/services/emailService.test.js (hunk 5 above)
Comment (Danyil-man) [reply src/services/emailService.test.js]: fixed

File: src/services/EmailService.js [hunk 6]
```diff
@@ -1,12 +1,17 @@
...
 
 const emailService = values => {
-  emailjs.send(
-    "service_hxod138",
-    "template_1k257d8",
-    values,
-    "hIa6EbQItOhL_Sxmg",
-  );
+  const isValid = isValidEmail(values.email);
+  if (isValide) {
```
Comment (akuzmichov-amplify) [inline src/services/EmailService.js]: Typo

**Review by MelenkoOleksandr — APPROVED** [review]

**Pull request merged**